app/assets/stylesheets/bootstrap-ui.scss

```diff
@@ -1 +1,32 @@
-@import 'bootstrap';
+@import 'bootstrap/functions';
+@import 'bootstrap/variables';
+@import 'bootstrap/mixins';
+@import 'bootstrap/root';
+@import 'bootstrap/images';
+@import 'bootstrap/grid';
+@import 'bootstrap/tables';
+@import 'bootstrap/forms';
+@import 'bootstrap/buttons';
+@import 'bootstrap/transitions';
+@import 'bootstrap/dropdown';
+@import 'bootstrap/button-group';
+@import 'bootstrap/input-group';
+@import 'bootstrap/custom-forms';
+@import 'bootstrap/nav';
+@import 'bootstrap/navbar';
+@import 'bootstrap/card';
+@import 'bootstrap/breadcrumb';
+@import 'bootstrap/pagination';
+@import 'bootstrap/badge';
+@import 'bootstrap/jumbotron';
+@import 'bootstrap/alert';
+@import 'bootstrap/progress';
+@import 'bootstrap/media';
+@import 'bootstrap/list-group';
+@import 'bootstrap/close';
+@import 'bootstrap/modal';
+@import 'bootstrap/tooltip';
+@import 'bootstrap/popover';
+@import 'bootstrap/carousel';
+@import 'bootstrap/utilities';
+@import 'bootstrap/print';
```

Comment:
Does this reduce the weird changes bootstrap makes?

Reply:
@teceler did we get the changeset small enough that we can put this back to @bootstrap instead?

Reply:
I feel like it's much cleaner to not pull in reboot and code and type, which do the things we'd have to override and have Strong Feelings About Things, but I can be swayed possibly.

Reply:
As long as we manually check new PRs with bootstrap components until this is fixed, this'll do for an MVP